Exiled.API/Features/Player.cs

```diff
@@ -739,11 +739,26 @@ public float MaxArtificialHealth
             }
         }
 
+        /// <summary>
+        /// Gets or sets the player's Hume Shield.
+        /// </summary>
+        /// <remarks>This value can bypass <see cref="FpcRole.MaxHumeShield"/>. However, this value will only be visible to the end-player as Hume Shield if <see cref="FpcRole.IsHumeShieldedRole"/> is <see langword="true"/>. Otherwise, the game will treat the player as though they have the amount of Hume Shield specified, even though they cannot see it.</remarks>
+        public float HumeShield
```

Comment:
Define an interface for the HumeShield like the game does (`IHumeShieldedRole`) and implement it only in classes that are able to show it in the HUD. We did a similar thing with the `ISubroutinedScpRole` interface.

If you want to wrap their HumeShieldModule, you can create the "HumeShieldedRole" class which implements the IHumeShieldedRole interface, then make all role that can see the hume shield inherit from it.